metrics_utility/automation_controller_billing/dataframe_engine/base.py

```diff
@@ -156,6 +160,15 @@ def unique_index_columns():
     def data_columns():
         pass
 
+    @staticmethod
+    def default_data_column_headers(file_name):
+        with tempfile.TemporaryDirectory(prefix='automation_controller_billing_data_') as temp_dir:
+            if os.path.exists(os.path.join(temp_dir, f'{file_name}')):
+                row1 = pd.read_csv(os.path.join(temp_dir, f'{file_name}'), nrows=0)
+                return pd.DataFrame(columns=row1)
+            else:
+                return pd.DataFrame([{}])
+
```

Comment:
this will never return anything other than an empty dataframe - there is no csv in a freshly created temp dir, and we should not assume there would be one anywhere.

However, this is only called from the dataframe classes, and they each have a `data_columns` method...